src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -28,7 +28,7 @@ public CancellationTest(ITestOutputHelper output)
         [InlineData(false, true)]
         [InlineData(true, false)]
         [InlineData(true, true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19038")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: dotnet/corefx #19038")]
```

Comment:
>I think I had the same question before: why aren't we using ActiveIssue anymore?

I think we are suing SkipOnTargetFramework instead of ActiveIssue because only the former takes the TargetFrameworkMoniker which has more granular choices that those available using ActiveIssue?